app/com/linkedin/drelephant/spark/fetchers/SparkFetcher.scala

```diff
@@ -94,38 +92,41 @@ class SparkFetcher(fetcherConfigurationData: FetcherConfigurationData)
         Success(data)
       },
       e => {
-        logger.error(s"Failed fetching data for ${appId}", e)
+        logger.error(s"Failed fetching data for ${appId}")
```

Comment:
As discussed, retain the exception message and turn this into a warning message.